### PR TITLE
added the space after the slash for clear reading

### PR DESCRIPTION
**Pull request by mohitmishra786**
As in the whole document, there was space after the slash and before the slash, so I just did it for 2 more so that it should look good as per the whole document.

### CODE REVIEW

**Comment by mohitmishra786** [comment]
@googlebot I signed it!

**Comment by mohitmishra786** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogle%2Fcaliban%2Fpull%2F101) for more info**.

@googlebot I signed it!

File: .gitignore [hunk 1]
```diff
@@ -64,7 +64,7 @@ instance/
 .scrapy
 
 # Sphinx documentation
-docs/_build/
+docs/ _build /
```
Comment (sritchie) [inline .gitignore:67]: happy to take the first change, but does this change still pick up that path? Would you reverting before merge? Thank you for this!

Comment (mohitmishra786) [reply .gitignore:67]: I do not get your point can you please elaborate as I am new to this. I will be grateful if you will help me out by explaining why codecov rejected this pull request.

**Pull request closed without merging**